src/wizard/javascript/profiling-onboarding/react/1.install.md

```diff
@@ -0,0 +1,22 @@
+---
+name: React (JS Self Profiling)
+doc_link: https://docs.sentry.io/platforms/javascript/guides/react/profiling/
+support_level: beta
+type: framework
+---
+
+#### Install
+
+Because our browser profiling integration is built on top of the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it's in beta and will likely only move out once the official spec progresses and gains adoption. As with any beta package, there are risks involved in using it - see platform [status](https://chromestatus.com/feature/5170190448852992)
```

Comment:
```suggestion
## Install

Because our browser profiling integration is built on top of the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it's in beta and will likely only move out once the official spec progresses and gains adoption. As with any beta package, there are risks involved in using it - see platform [status](https://chromestatus.com/feature/5170190448852992).
```